src/components/SingleArticle.jsx

```diff
@@ -0,0 +1,53 @@
+import { useState, useEffect } from "react";
+import { getArticleById } from "../utils/axios";
+import { useParams } from "react-router-dom";
+import { Link } from "react-router-dom";
+
+export default function SingleArticle() {
+  const [article, setArticle] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    getArticleById(article_id)
+      .then((singleArticle) => {
+        setArticle(singleArticle);
+      })
+      .catch((err) => {
+        console.log(err, "ERROR");
```

Comment:
consider removing console.log? I feel like this is personal preference since it is in the catch block